docs/blog/2019-03-22-introducing-gatsby-preview-beta/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Introducing Gatsby Preview Beta
+date: 2019-03-22
+author: Marisa Morby
+excerpt: "Introducing the Gatsby Preview Beta: learn more and sign up today for early access to our first commerical product."
+tags:
+  - preview
+  - cloud
+  - commercial
+---
+
+## What we’ve been up to
+
+Back in July we were excited to send out our [alpha version of Gatsby Preview](https://www.gatsbyjs.org/blog/2018-07-17-announcing-gatsby-preview/). The team has been working hard getting your feedback and incorporating it into Gatsby Preview beta.
```

Comment:
I think this would benefit from a stronger, more active lead sentence (`excited` also fits great in the next paragraph):
```suggestion
Back in July we sent out our [alpha version of Gatsby Preview](https://www.gatsbyjs.org/blog/2018-07-17-announcing-gatsby-preview/). The team has been working hard getting your feedback and incorporating it into Gatsby Preview beta.
```